dev/io.openliberty.org.jboss.resteasy.validator.provider/resources/META-INF/services/javax.ws.rs.ext.Providers

```diff
@@ -0,0 +1,4 @@
+org.hibernate.validator.HibernateValidator
```

Comment:
Same as above - I wonder if we need this file at all. I think that when the beanValidation-3.0 feature is enabled, that will automatically supply a ValidationProvider for us.